.erb-lint.yml

```
@@ -6,7 +6,21 @@ linters:
     # issues that haven't been resolved yet. Fix 'em up!
     exclude:
       - '*/app/views/account_reset/*'
-      - '*/app/views/accounts/*'
+      - '*/app/views/accounts/_phone.html.erb'
+      - '*/app/views/accounts/_password_reset.html.erb'
+      - '*/app/views/accounts/_pending_profile_gpo.html.erb'
+      - '*/app/views/accounts/_backup_codes.html.erb'
+      - '*/app/views/accounts/_personal_key.html.erb'
+      - '*/app/views/accounts/_auth_apps.html.erb'
+      - '*/app/views/accounts/_identity_item.html.erb'
+      - '*/app/views/accounts/_webauthn.html.erb'
+      - '*/app/views/accounts/_piv_cac.html.erb'
+      - '*/app/views/accounts/show.html.erb'
+      - '*/app/views/accounts/actions/_manage_personal_key.html.erb'
+      - '*/app/views/accounts/_nav_auth.html.erb'
+      - '*/app/views/accounts/_pending_profile_bounced_gpo.html.erb'
+      - '*/app/views/accounts/_connected_app.html.erb'
+      - '*/app/views/accounts/_emails.html.erb'
```

nit: alphabetize this?

```suggestion
      - '*/app/views/accounts/_auth_apps.html.erb'
      - '*/app/views/accounts/_backup_codes.html.erb'
      - '*/app/views/accounts/_connected_app.html.erb'
      - '*/app/views/accounts/_emails.html.erb'
      - '*/app/views/accounts/_identity_item.html.erb'
      - '*/app/views/accounts/_nav_auth.html.erb'
      - '*/app/views/accounts/_password_reset.html.erb'
      - '*/app/views/accounts/_pending_profile_bounced_gpo.html.erb'
      - '*/app/views/accounts/_pending_profile_gpo.html.erb'
      - '*/app/views/accounts/_personal_key.html.erb'
      - '*/app/views/accounts/_phone.html.erb'
      - '*/app/views/accounts/_piv_cac.html.erb'
      - '*/app/views/accounts/_webauthn.html.erb'
      - '*/app/views/accounts/actions/_manage_personal_key.html.erb'
      - '*/app/views/accounts/show.html.erb'
```

---

> nit: alphabetize this?

Good idea 👍